lldb/source/API/SBThread.cpp

```diff
@@ -138,13 +139,13 @@ StopReason SBThread::GetStopReason() {
 
   StopReason reason = eStopReasonInvalid;
```

Comment:
Too bad we don't have an eStoppedReasonNotStopped.  That is probably a little too tongue-in-cheek, however...